cmake/modules/AddSwift.cmake

```diff
@@ -730,7 +730,8 @@ function(_add_swift_library_single target name)
         OBJECT_LIBRARY
         SHARED
         STATIC
-        TARGET_LIBRARY)
+        TARGET_LIBRARY
+        INSTALL_WITH_SHARED)
```

Comment:
I'm not sure I agree with adding yet another single purpose flag to `add_swift_target_library`. Is there a more generic purpose here? Say maybe an `INSTALL_PATH` or something?

Reply:
Yeah, I'm torn about that too. YAGNI convinced me to stick with just the flag for now. A small generalization in between this and INSTALL_PATH might be to have an `INSTALL_AS [SHARED|STATIC]` flag, so you could also choose to send dylibs to the static dir and v.v.?